domain/src/test/java/org/oppia/android/domain/profile/ProfileManagementControllerTest.kt

```diff
@@ -532,6 +532,32 @@ class ProfileManagementControllerTest {
     val profile = monitorFactory.waitForNextSuccessfulResult(profileProvider)
     assertThat(profileManagementController.getCurrentProfileId().internalId).isEqualTo(2)
     assertThat(profile.lastLoggedInTimestampMs).isNotEqualTo(0)
+    assertThat(profile.numberOfLogins).isNotEqualTo(0)
+  }
+
+  @Test
+  fun testLoginToProfile_addProfile_loginToProfile_logoutProfile_checkNumberOfLoginsIsCorrect() {
```

Comment:
This is potentially verifying more than one thing. Per my comment above, perhaps all we need to check here is that logging in, then out, then in results in a login count of 2? '0' and '1' cases can be tested in separate cases per the above.

That ensures the one-behavior-per-test convention is kept.

Reply:
This is how one-behaviour-per-test convention is being tried to follow in my latest commit: 

1) Check no_of_logins = 0 when profile is just created: **testAddProfile_addProfile_checkProfileIsAdded**
2) Check no_of_logins = 1 after first login: **testLoginProfile_addedProfile_profileIdTimestampAndNumberOfLoginsCorrectlyDefaulted**

3) Check no_of_logins after two logins: **testLoginToProfile_addProfile_loginToProfileTwice_checkNumberOfLoginsIsTwo**

Reply:
please let me know I still need to add more tests here

Reply:
I think the latest state is now clear, though I suggested a different name for 'defaulted' (since my understanding before might've been incorrect given the confusion of when the login count was actually meant to be incremented in each test case).